app/src/main/res/layout/fragment_comments.xml

```diff
@@ -49,6 +49,17 @@
 
     </LinearLayout>
 
+    <TextView
+        android:id="@+id/comments_disabled"
+        android:layout_width="wrap_content"
+        android:layout_height="wrap_content"
+        android:layout_below="@id/empty_state_view"
+        android:layout_centerHorizontal="true"
+        android:text="@string/comments_are_disabled"
+        android:textSize="20sp"
+        android:visibility="gone"
+        tools:visibility="visible" />
+
```

Comment:
I would move this at the bottom of the `empty_state_view` layout. It shouldn't change anything theoretically, but I remember some scrolling issues that happened after adding the error panel here, and `empty_state_view` seems like a suitable place anyway.

Or another idea: maybe you could just change the text of the "No comments" `TextView` to "Comments are disabled" instead of using two `TextView`s. Saying that comments are disabled implies that there are no comments.

Reply:
> Or another idea: maybe you could just change the text of the "No comments" TextView to "Comments are disabled" instead of using two TextViews. Saying that comments are disabled implies that there are no comments.

Yeah that's definitely the better idea.

Updated it, now it looks like this:
![grafik](https://user-images.githubusercontent.com/40789489/122643301-54104880-d10f-11eb-88b0-62f773236a5c.png)

Reply:
Great